custom_components/tuya_local/devices/simple_contact_sensor.yaml

```diff
@@ -0,0 +1,18 @@
+name: Simple (door/window) sensor
+products:
+  - id: 0mjj67aaztcfmc3i
+    name: AW201_XR3
+primary_entity:
+  entity: cover
```

Comment:
If it is just a sensor, then it should be implemented as a binary_sensor.
cover is for control of curtains/doors etc.